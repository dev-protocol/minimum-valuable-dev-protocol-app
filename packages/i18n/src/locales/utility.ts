import { promises as fs } from 'fs'

const basePath = `${__dirname}/../../../web/public/locales`

export const makeLocaleJson = async (filePath: string, data: any) =>
  fs.writeFile(`${basePath}/${filePath}`, JSON.stringify(data))
