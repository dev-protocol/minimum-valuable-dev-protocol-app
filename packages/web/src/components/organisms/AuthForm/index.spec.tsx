import React from 'react'
import { render } from '@testing-library/react'
import { AuthForm } from '.'
import 'src/__mocks__/window/matchMedia.mock'
import { useEffectAsync } from 'src/fixtures/utility'

jest.mock('src/fixtures/utility')
jest.mock('src/fixtures/dev-kit/hooks')

describe(`${AuthForm.name}`, () => {
  test('Snapshot', () => {
    ;(useEffectAsync as jest.Mock).mockImplementation(() => {})
    const component = render(<AuthForm property="property" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
