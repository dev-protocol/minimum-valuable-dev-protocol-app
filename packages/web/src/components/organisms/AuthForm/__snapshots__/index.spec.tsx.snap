// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthForm Snapshot 1`] = `
<body>
  <div>
    <div
      class="bja7zw-0 bVsGvT"
    >
      <div
        class="bja7zw-1 cmaEXV"
      >
        <span>
          Associating Property:
        </span>
        <span>
          property
        </span>
      </div>
      <div
        class="bja7zw-1 cmaEXV"
      >
        <span>
          Arguments:
        </span>
        <form
          class="ant-form ant-form-horizontal"
          id="basic"
        >
          <button
            class="ant-btn ant-btn-primary"
            type="submit"
          >
            <span>
              Authenticate
            </span>
          </button>
        </form>
      </div>
    </div>
  </div>
</body>
`;
